webapp/src/main/webapp/templates/freemarker/lib/lib-microformats.ftl

```diff
@@ -16,7 +16,7 @@
 			<#assign mst = type />
 			<#break>
 		</#list>
-    	<@getItemType mst "infoContentEntity"/>
+       <@getItemType mst!"" "infoContentEntity"/>
```

Comment:
Probably this should be mst!"Document", and line 14 in this file should be removed.

Reply:
Thanks, I have made the change.